README.md

```diff
@@ -1,5 +1,6 @@
 # Tizen RT
 
+[![Build Status](https://travis-ci.org/Samsung/TizenRT.svg?branch=master)](https://travis-ci.org/Samsung/TizenRT)
```

Comment:
Let's move dynamic to behind static. And in my opinion, license is more important than build status.